source/develop/release-management/features/network/manageiq_ovn.html.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Integrating OVN to ManageIQ
+category: feature
+authors: Alona Kaplan
+wiki_category: Feature
+wiki_title: Integrating OVN to ManageIQ
+wiki_revision_count: 1
+wiki_last_updated: 2017-07-09
+feature_name: Integrating OVN to ManageIQ
+feature_modules: manageiq_providers_ovirt - infra_manager, network_manager, manageiq-ui-classic - network related controllers
+feature_status: In Development
+---
+
+# Integrating OVN to ManageIQ
+
+## Summary
+Discovering OVN as the network provider of oVirt and adding the ability to show, add, update and remove networks, subnets and ports directly from ovirt-provider-ovn.
+
+## Owner
+
+*   Name: [Alona Kaplan](https://github.com/AlonaKaplan)
+
+*   Email: <alkaplan@redhat.com>
+
+## Terminology
+* OVN: Open Virtual Network. OVN is an OVS (Open vSwitch) extension that brings Software Defined Networking to OVS.
+* oVirt OVN provider: is a proxy that interacts with OVN. The provider has 'neutron like' API.
+* ManageIQ: is a cloud management platform that allows centralized management of various virtualization, private cloud, public cloud, containers, middleware and software defined networking technologies.
+
+## Detailed Description
+ManageIQ has an infra provider for oVirt. The feature adds automatic discovery of oVirt's network provider (OVN provider) once the infra provider is added/refreshed.
+ManageIQ users will be able to perform CRUD actions on networks, subnets and ports directly on the OVN provider without passing through oVirt.
```

Comment:
why do we need to CRD ports? removing a port is going to spell trouble when we attempt to re-run the VM using it.

Reply:
done